### ISSUE TITLE
Font family config option

### ISSUE THREAD

**Issue by volvofixthis** [moverest/wl-kbptr#12]
Could you add configuration option for font family please? I use terminus font and want have ability to change default one.

**Comment by moverest** [comment]
Yeah I could add that.

The reason I didn't bother with it before is that I configure fonts on my system with fontconfig; in `~/.config/fontconfig/fonts.conf `:

```xml
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
  <match target="pattern">
    <test name="family"><string>serif</string></test>
    <edit name="family" mode="prepend"><string>IBM Plex Serif</string></edit>
  </match>

  <match target="pattern">
    <test name="family"><string>sans-serif</string></test>
    <edit name="family" mode="prepend"><string>IBM Plex Sans</string></edit>
  </match>

  <alias>
    <family>monospace</family>
    <prefer>
      <family>IBM Plex Mono</family>
      <family>Noto Color Emoji</family>
      <family>Noto Emoji</family>
    </prefer>
  </alias>
</fontconfig>
```

This enables me to set the font all over the system and set all the programs to use `serif`, `sans-serif` and `monospace`. Feel free to try this in the meantime.

**Comment by moverest** [comment]
Now implemented with b7e0d7326a3a72eb716e66a7900d23dd396c53bf.
